src/community/backlog/index.md.njk

```diff
@@ -18,13 +18,20 @@ Here is a list of the components, patterns and updates currently on the Design S
 
 
 <table class="govuk-table app-table--constrained">
-  <tbody><tr>
-    <th class="govuk-table__header">Name</th>
-    <th class="govuk-table__header" style="width: 90px; text-align: right">Status</th>
-  </tr>
-
-
+  <thead>
```

Comment:
This is a good improvement :+1: Do you think it's worth splitting into a separate commit? It's not really covered by the current commit message.

Reply:
Split out

Reply:
👏